server.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const app = express();
```

Comment:
Keep your import statement separate from your application logic for better readability:

```js
const express = require("express");
const mongoose = require("mongoose");
const bodyParser = require("body-parser");
require("dotenv").config();

const User = require("./models/User");

const app = express();
app.use(bodyParser.urlencoded({ extended: true }));
app.use(bodyParser.json());
```